browser-test/src/applicant_application_index.test.ts

```diff
@@ -485,8 +485,6 @@ describe('applicant program index page with images', () => {
     await validateAccessibility(page)
   })
 
-  // TODO(#5676): Test with a very small image.
```

Comment:
I've tested manually with a very small image and it gets scaled up, which is WAI. I don't think we need a browser test for it.